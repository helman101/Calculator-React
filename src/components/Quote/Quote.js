import React from 'react';
import Navbar from '../Navbar/Navbar';
import * as styles from '../../style.module.css';

const Quote = () => (
  <div>
    <Navbar />
    <div className={`${styles.width100} ${styles.pages} ${styles.quote}`}>
      <p>Without mathematics, there’s nothing you can do.</p>
      <p>Everything around you is mathematics.</p>
      <p>Everything around you is numbers.</p>
      <p>– Shakuntala Devi</p>
    </div>
  </div>
);

export default Quote;
