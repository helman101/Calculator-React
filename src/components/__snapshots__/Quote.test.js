import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote/Quote';

it('renders correctly', () => {
  const quotePage = renderer
    .create(<Quote />)
    .toJSON();
  expect(quotePage).toMatchSnapshot();
});
