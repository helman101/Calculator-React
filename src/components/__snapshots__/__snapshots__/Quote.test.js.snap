// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="undefined undefined undefined"
>
  <p>
    Without mathematics, there’s nothing you can do.
  </p>
  <p>
    Everything around you is mathematics.
  </p>
  <p>
    Everything around you is numbers.
  </p>
  <p>
    – Shakuntala Devi
  </p>
</div>
`;
